enable/savage/trait_defs/qt/svg_button_editor.py

```diff
@@ -15,10 +15,13 @@
 # -----------------------------------------------------------------------------
 import os.path
 
+from pyface.qt import QtCore, QtGui
 from traits.api import Bool, Any, Str
-from traitsui.qt4.editor import Editor
+try:
+    from traitsui.qt.editor import Editor
+except ModuleNotFoundError:
+    from traitsui.qt4.editor import Editor
```

Comment:
Doesn't this PR's change force us to use a `traitsui` which _doesn't_ have a `qt4` subpackage?

Reply:
No, it's currently backward compatible because it was low effort to do that (only 3-4 places needed try/excepts to achieve it) and it simplifies releasing.